check_img_lazy.py

```diff
@@ -0,0 +1,61 @@
+import os
+import sys
+import subprocess
+from bs4 import BeautifulSoup
+
+def build_website():
+    """Build the website using the specified command."""
+    try:
+        subprocess.check_call(['./hack/docker/test.sh'])  
+    except subprocess.CalledProcessError as e:
+        print(f"Website build failed: {e}")
+        sys.exit(1)
```

Comment:
Please leave this part out of the script.

This script will eventually run in a GitHub Actions workflow:
- Build the website
- Run the script
  - If there are images w/o `loading=lazy`, script should fail
  - If there are no images, script should exit successfully

Can you put the script in a GitHub actions workflow please?

Also, we need to document this change with instructions. In the relevant section here: https://github.com/knative/docs/blob/main/contribute-to-docs/README.md

Reply:
Yeah, sure, I will remove the build function from the script.
I will look into the GitHub action workflow and add the script as soon as possible.